src/tools/qt-gui/src/main.cpp

```diff
@@ -12,15 +12,19 @@
 #include "confignode.hpp"
 #include "undomanager.hpp"
 #include "guibackend.hpp"
+#include "guisettings.hpp"
 
 int main(int argc, char* argv[])
 {
 	QApplication app(argc, argv);
+	//keynames are only split at unescaped slashes
+	app.setProperty("KEY_DELIMITER", QRegularExpression("(?<!\\\\)/"));
```

Comment:
Hi,

0003088 wrote:
> Could you give me a hint where to find the iterator? I was not able to
> locate it.

It was not there, thats why I asked if you need it.
I now added a bidirectional (reverse) iterator in key.hpp
d3ab632e209e7df2f3dba1f1a39a2ed234c24c8e

Tests are here
src/bindings/cpp/tests/testcpp11_iter_name.cpp

You find examples how to use it in
src/bindings/cpp/examples/cpp_example_iter_name.cpp

best regards